lighthouse-core/gather/driver/execution-context.js

```diff
@@ -240,7 +240,7 @@ class ExecutionContext {
    * @return {string}
    */
   static serializeArguments(args) {
-    return args.map(arg => arg === undefined ? 'undefined' : JSON.stringify(arg)).join(',');
+    return args.map(arg => arg === undefined ? 'undefined' : JSON.stringify(arg)).join(', ');
```

Comment:
Why is this changing? Drive by style change? Not sure what it has to do with the report

Reply:
@paulirish .

Reply:
oh, I didn't see `build-report-components.js` is actually using `ExecutionContext.serializeArguments`. Could it just have its own one liner? :)

Reply:
It's sufficiently complex to be reused. I tried just rewriting from memory but the undefined bit tripped me up. granted, `undefined` is no longer used in this use case.

Reply:
> It's sufficiently complex to be reused. I tried just rewriting from memory but the undefined bit tripped me up. granted, `undefined` is no longer used in this use case.

This does seem like it might be over DRYing.

Without the `undefined` it's `const argsSerialzed = args.map(JSON.stringify).join(', ')` (and really if there are characters that `JSON.stringify` is actually escaping here, our tagNames and classNames are going to have a bad time. Probably better safe than sorry, though).

The duplication is small and it avoids a dependency between the build system and a non-public function in core gathering, which seems like a win.